server/game/cards/12-KotI/DesertRaider.js

```diff
@@ -0,0 +1,52 @@
+const DrawCard = require('../../drawcard.js');
+const ChallengeTypes = require('../../ChallengeTypes');
+
+class DesertRaider extends DrawCard {
+    setupCardAbilities() {
+        this.reaction({
+            when: {
+                afterChallenge: event => event.challenge.loser === this.controller && this.controller.canPutIntoPlay(this)
+            },
+            location: 'dead pile',
+            target: {
+                type: 'select',
```

Comment:
The card explicitly says choose, so this isn't needed. `type: 'select'` is more for implicit targeting (e.g. "Place a poison token on a character without intrigue icon")